cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -85,7 +88,7 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 				id = seqValue + 1;
 			}
 			else {
-				id = START_VALUE;
+				id = Integer.parseInt(startValue);
```

Comment:
In the UI you allow to specify project variables, so you should do
`startValue = getExecutionContext().getVariables().replaceVariables(startValue);`
beforehand.

Reply:
How do you think this was resolved? I don't see a change.